dojo/tools/awssecurityhub/parser.py

```diff
@@ -81,11 +81,30 @@ def get_item(finding: dict, test):
                     mitigated = datetime.strptime(finding.get("LastObservedAt"), "%Y-%m-%dT%H:%M:%fZ")
             else:
                 mitigated = datetime.utcnow()
-
+    elif aws_scanner_type == "GuardDuty":
+        mitigations = finding.get("FindingProviderFields", {}).get("Types")
+        for mitigate in mitigations:
+            mitigation += mitigate + "\n"
+        mitigation += "https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_finding-types-active.html"
+        active = True
+        if finding.get("RecordState") == "ACTIVE":
+            is_Mitigated = False
+        else:
+            is_Mitigated = True
+        if finding.get("LastObservedAt", None):
```

Comment:
Looks like this should be nested under the `else` on line 92, otherwise every Finding will have a `mitigated` even if `is_Mitigated` is `False`

Reply:
I changed it according to your comment.

The problem here is that Guard Duty is a Threat Detection Service in AWS.

Thus, it can be like this: The issue does not occur at this point of time anymore, but is also not mitigated as there was an attack which still needs doings.
Thus "LastObservedAt" is rather that the attack is not happening anymore, but is also not resolved.